omod/src/main/webapp/resources/scripts/conditionlist/controllers/addcondition.controller.js

```diff
@@ -21,7 +21,7 @@ function ConditionController($scope, RestfulService, ConditionModel, ConceptMode
      * @type {Function}
      */
     self.saveCondition = self.saveCondition || function () {
-        if($scope.condition.status == INACTIVE_STATUS && $scope.condition.onSetDate > $scope.condition.endDate){
+        if($scope.condition.status == INACTIVE_STATUS && Date.parse($scope.condition.onSetDate) > Date.parse($scope.condition.endDate)){
```

Comment:
How is the above change related to the reported problem?

Reply:
Since I have changed the date object to the UTCstring...this condition could not evaluate as intended thus the need to parse the string back to date object in the condition check